wcfsetup/install/files/lib/system/language/event/CopyLanguageContent.class.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace wcf\system\language\event;
+
+use wcf\data\language\Language;
+use wcf\system\event\IEvent;
+
+/**
+ * Indicates that the contents of a language should be copied to another one.
+ *
+ * @author Florian Gail
+ * @copyright 2001-2023 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since 6.0
+ */
+final class CopyLanguageContent implements IEvent
```

Comment:
I realize it just now, but the event name does not follow our (I believe not yet documented) naming scheme. It should be: `LanguageContentCopying`.

The pattern is:
- The prefix describes the “object” that is affected (LanguageContent).
- The suffix describes the action that is happening (“Copy”)
- The tense (as in “past tense”) describes whether the operation is in progress or already happened (“-ing” = in progress).